code/resources/js/vuescript.js

```diff
@@ -0,0 +1,23 @@
+import Navigation from "./components/Navigation"
+import Home from "./components/Home"
+import Footer from "./components/Footer"
+import Login from "./components/Login"
+
+var vue = new Vue({
+    el: '#app',
+    components: {
+        Navigation,
+        Home,
+        Footer,
+        Login
+    },
+    data: function () {
+        return {
+        seen: false
+        }
+    },
+        
+
+    template: '<div><navigation @seen="seen = !seen"/><login :seen="seen"/><home></home><Footer></Footer></div>'
```

Comment:
mysle ze mozesz sobie stworzyc jeden glowny komponent aplikacji, na przyklad `App.vue` i tam zaimportowac te komponenty. Niech ten plik odpowiada tylko za stworzenie aplikacji we vue.

Reply:
Nie do konca wiem czemu mam tylko background color widoczny :/ 
To moj plik js
```
import AppVue from "./components/App"

var vue = new Vue({
    el: '#app',
    components: {
        AppVue
    },
    data: function () {
        return {
        seen: false
        }
    },
        
    template: '<div><appvue/></div>'
                
})
```
To moj plik vue
```

<template>
    <div>
        <navigation @seen="seen = !seen"/>
        <login :seen="seen"/>
        <home/>
        <Footer/> 
    </div>
</template>
<script>
import Navigation from "./Navigation"
import Home from "./Home"
import Footer from "./Footer"
import Login from "./Login"

export default {
    props: {
       seen: {
        type: Boolean,
        required: true,
        
       }
      }
}
</script>
<style>

</style>
```